src/components/Motorcycles/Motorcycle/Motorcycle.js

```diff
@@ -53,12 +54,14 @@ function Motorcycle(props) {
             <p className={styles.description}>
               {props.description}
             </p>
+            {auth 
+              ? <p className="mt-2">Dostępność: 2 motocykle</p>
+              : <p className="mt-2">Dostępność: zaloguj</p>
+            }

```

Comment:
Always try to put inline if as low as possible to render the smallest part of the html. So in this case should be 

<p className="mt-2">{  auth ? 'Dostępność: 2 motocykle' : 'Dostępność: zaloguj'  }</p>